### PR TITLE
docs: Move contents from DraftUserGuide to UserGuide

### PR DESCRIPTION
**Pull request by jiakai-17**
## What type of PR is this? (check all applicable)

-   [ ] Refactor
-   [ ] Feature
-   [ ] Bug Fix
-   [ ] Optimization
-   [x] Documentation Update

## Description

* Move contents from DraftUserGuide to UserGuide
* Replace introduction
* Replace Admonitions
* Rephrase sections in Quick Start
* Update Release URL in Quick Start
* Add page breaks
* Add findPerson and findTask sections
* Add Command Summary
* Delete DraftUserGuide

## Related Tickets & Documents

-   Related Issue #38 

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 75% and above for new changes._

-   [ ] Yes
-   [x] No, and this is why: See deployment here: https://docs-jiakai-move-ug-changes.coordimate.pages.dev/
-   [ ] No, there are UI changes that I'm unable to cover.
-   [ ] I need help with writing tests

### CODE REVIEW

**Review by HugeNoob — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,26 +3,40 @@ layout: page
...
+4. Open a command terminal, `cd` into the folder you put `addressbook.jar` in, and use the `java -jar addressbook.jar` command to start CoordiMate.<br>
+
+   A GUI similar to the below picture should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
    * `list` : Lists all contacts.
```
Comment (pra-navi) [inline docs/UserGuide.md:42]: could you changes these sample commands to listPerson, addPerson etc

File: docs/UserGuide.md [hunk 2]
```diff
@@ -35,33 +49,89 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
+## Usage
+
+{% include admonition.html type="info" title="Info: About Command Formats" body="
+
+<ul>
+  <li>
+    <p>
+      Words in <code>UPPER_CASE</code> are the parameters to be supplied by the user.<br>
+      e.g. in <code>add n/NAME</code>, <code>NAME</code> is a parameter which can be used as <code>add n/John Doe</code>, <br>
```
Comment (m1oojv) [inline docs/UserGuide.md:91]: I think the commands here also need updating

**Review by m1oojv — CHANGES_REQUESTED** [review]

**Review by m1oojv — APPROVED** [review]
LGTM Nice work

**Pull request merged**